packages/react/src/cloud/auiV0.ts

```
@@ -0,0 +1,130 @@
+      switch (type) {
+        case "text": {
+          return {
+            type: "text",
+            text: part.text,
+          };
+        }
+
+        case "tool-call": {
+          if (!isJSONValue(part.result)) {
+            console.warn(
+              "tool-call result is not JSON! " + JSON.stringify(part),
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)